src/redux/sagas/listitem.saga.js

```diff
@@ -29,7 +29,7 @@ function* getList(action) {
         url: `/api/list/`,
         data: action.payload
 })
-    console.log('back from GET with:', response.data);
+    //console.log('back from GET with:', response.data);
```

Comment:
If this log isn't in use, why not just delete it?

Reply:
deleted all console.logs